### PR TITLE
[IA] Move python automatic page to zero-code

### PR DESCRIPTION
**Pull request by svrnm**
Contributes to https://github.com/open-telemetry/opentelemetry.io/issues/4427

### CODE REVIEW

**Review by theletterf — APPROVED** [review]

File: content/en/docs/languages/python/libraries.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ You might also want to configure an appropriate exporter to
 telemetry backends.
 
 You can also check the
-[automatic instrumentation for Python](/docs/languages/python/automatic).
+[automatic instrumentation for Python](/docs/zero-code/python/).
```
Comment (xrmx) [inline content/en/docs/languages/python/libraries.md:107]:
```suggestion
[Zero-code instrumentation for Python](/docs/zero-code/python/).
```

**Comment by theletterf** [comment]
@xrmx PTAL

**Comment by lzchen** [comment]
Is the scope of this decision to require all SIGs to rebrand to `zero-code` or is this simply a documentation change on the `io` website?

**Comment by svrnm** [comment]
> Is the scope of this decision to require all SIGs to rebrand to autoinstrumentation and it's equivalents to `zero-code` or is this simply a documentation change on the `io` website?

For now it is the second: an information architecture change to separate API&SDK and any kind of "automation/zero-code" around it.

I would love to see the first happening as well (and personally I am not attached to "zero-code", but I prefer it over "automatic", what I want is a consistent term), but this is blocked due to the fact that we [have no process for consistent terminology across the project](https://github.com/open-telemetry/community/issues/2165) and discussions [like this one](https://github.com/open-telemetry/opentelemetry.io/discussions/3809) although promoted through various forums never got any attention outside docs.

**Review by lzchen — APPROVED** [review]

**Pull request merged**